### PR TITLE
Add remove promotion rule and action cookbooks

### PR DESCRIPTION
**Pull request by tophsic**
| Q               | A
| --------------- | -----
| Branch?         | master
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | 
| License         | MIT

Documentation proposition to explain how to remove a promotion rule or a promotion action.

### CODE REVIEW

**Comment by tophsic** [comment]
Promotion rules and actions can be also unregistered from to registries (`src/Sylius/Bundle/PromotionBundle/DependencyInjection/Compiler/RegisterRuleCheckersPass.php`). I will look at it later.

File: docs/cookbook/promotions/remove-promotion-action.rst [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+    <?php
+
+    namespace App;
+
+    use App\DependencyInjection\Compiler\PromotionPass;
+
+    final class Kernel extends BaseKernel
+    {
+    // ...
```
Comment (Zales0123) [inline docs/cookbook/promotions/remove-promotion-action.rst]: Indent

File: docs/cookbook/promotions/remove-promotion-action.rst [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+.. code-block:: php
+
+    <?php
+
+    namespace App\DependencyInjection\Compiler;
+
+    use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+    use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+    class PromotionPass implements CompilerPassInterface
```
Comment (Zales0123) [inline docs/cookbook/promotions/remove-promotion-action.rst]: Missing `final` keyword

Comment (Zales0123) [reply docs/cookbook/promotions/remove-promotion-action.rst]: How about `PromotionActionsPass`?

File: docs/cookbook/promotions/remove-promotion-action.rst [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    {
+        public function process(ContainerBuilder $container): void
+        {
+            $parameter = $container->getParameter('sylius.promotion_actions');
+
+            $actionsToRemove = [
+                'sylius.promotion_action.shipping_percentage_discount'
```
Comment (Zales0123) [inline docs/cookbook/promotions/remove-promotion-action.rst]: Missing coma at the end of line

Comment (Zales0123) [reply docs/cookbook/promotions/remove-promotion-action.rst]: Maybe we could suggest removing promotions actions by their **type**? Then we could have a simple array like:

```
$actionsToRemove = [
    'shipping_percentage_discount',
];
```

And there would be no need to get the definition from the container :)

File: docs/cookbook/promotions/remove-promotion-action.rst (hunk 2 above)
Comment (Zales0123) [reply docs/cookbook/promotions/remove-promotion-action.rst]: This also refers to the next file

File: docs/cookbook/promotions/remove-promotion-action.rst (hunk 3 above)
Comment (Zales0123) [reply docs/cookbook/promotions/remove-promotion-action.rst]: This also refers to the next file

Comment (tophsic) [reply docs/cookbook/promotions/remove-promotion-action.rst]: Thx for your review. I amend modifications to my commit.

**Pull request closed without merging**